### PR TITLE
Added a Town Hall

### PR DESCRIPTION
**Pull request by FarkasKBence**
#### Summary
 ```SUMMARY: Content Added a Town Hall```

#### Purpose of change
Wanted to add a multitle city building.
Also it's a government building.

#### Describe the solution
I made 2x2x2 town hall.  (2x2x3 including the roof)
It has a(n):
*  Large room for town hall meetings
* Office
* Information desk
* Statues about the town's history 
* 2 rooms for educational purposes
* 1 small ballroom 
* Flat roof 

#### Screenshots
Town_hall_1

Town_hall_2
Town_hall_3

### CODE REVIEW

**Comment by chaisawlajatang** [comment]
Wow, that's just gorgeous.

**Comment by nexusmrsep** [comment]
> Wow, that's just gorgeous.

My words exactly

**Comment by I-am-Erk** [comment]
I would suggest putting some f_air_conditioner up on the roof, maybe in blocks of 4, from this pr: https://github.com/CleverRaven/Cataclysm-DDA/pull/28192 - it should be merged very soon here. Also a roof of a large building like this might have a few pipes and standing tanks for water.

A city building might have quite a few f_filing_cabinets as well.

I'll take a closer look at the maps themselves soon, looking really good.

File: data/json/regional_map_settings.json [hunk 1]
```diff
@@ -550,6 +550,7 @@
         "publicgarden": 200
       },
       "shops": {
+        "town_hall": 150,
```
Comment (ZhilkinSerg) [inline data/json/regional_map_settings.json]:
```suggestion
        "town_hall": 150,
```

File: data/json/mapgen/town_hall [hunk 2]
```diff
@@ -0,0 +1,267 @@
+[
```
Comment (ZhilkinSerg) [inline data/json/mapgen/town_hall]: File should be renamed to `town_hall.json`.

**Pull request merged**